DOCUMENTATION.md

```diff
@@ -1,810 +1,807 @@
 
-#### `template_id`
-Find by clicking **API info** for the template you want to send.
+You can only get the status of messages that are 7 days old or less.
 
-#### `reference`
-An optional identifier you generate. The `reference` can be used as a unique reference for the notification. Because Notify does not require this reference to be unique you could also use this reference to identify a batch or group of notifications.
+## Status - text and email
 
-You can omit this argument if you do not require a reference for the notification.
+### Created
 
-#### `personalisation`
-If the template has placeholders you need to provide their values as a Hash, for example:
+The message is queued to be sent to the provider. The notification usually remains in this state for a few seconds.
 
-```ruby
-  personalisation: {
-    'first_name' => 'Amala',
-    'reference_number' => '300241',
-  }
-```
+### Sending
 
-You can omit this argument if the template does not contain placeholders and is for email or sms.
+The message is queued to be sent by the provider to the recipient, and GOV.UK Notify are waiting for delivery information.
 
-#### `personalisation` (for letters)
+### Pending (text message only)
 
-If you are sending a letter, you will need to provide the letter fields in the format `"address_line_#"`, for example:
+GOV.UK Notify received a callback from the provider but the device has not yet responded. Another callback from the provider will determine the final status of the notification.
 
-```ruby
-personalisation: {
-    'address_line_1' => 'The Occupier',
-    'address_line_2' => '123 High Street',
-    'address_line_3' => 'London',
-    'postcode' => 'SW14 6BH',
-    'first_name' => 'Amala',
-    'reference_number' => '300241',
-}
-```
+### Delivered
 
-The fields `address_line_1`, `address_line_2` and `postcode` are required.
+The message was successfully delivered.
 
-</details>
+### Sent (text message only)
 
+The text message was delivered internationally. GOV.UK Notify may not receive additional status updates depending on the recipient's country and telecoms provider.
 
-## Get the status of one message
+### Failed
 
-#### Method
+This covers all failure statuses:
 
-<details>
-<summary>
-Click here to expand for more information.
-</summary>
+- `permanent-failure` - "The provider was unable to deliver message, email or phone number does not exist; remove this recipient from your list"
+- `temporary-failure` - "The provider was unable to deliver message, email inbox was full or phone was turned off; you can try to send the message again"
+- `technical-failure` - "Notify had a technical failure; you can try to send the message again"
 
-```ruby
-notification = client.get_notification(id) # => Notifications::Client::Notification
-```
+## Status - letter
 
-</details>
+### Failed
 
+The only failure status that applies to letters is `technical-failure`. Notify had an unexpected error while sending to our printing provider.
```

Comment:
Letters also have the failure state of `virus-scan-failed`. (But I can't remember if the plan was not mention that in these docs or whether it was going to be included)

Reply:
I think we said we were going to leave it out, after a discussion with Becca. We can check next week.